keyboards/arrowmechanics/wings/keymaps/default/keymap.c

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2024 Filip Strzałka <filip@arrowmechanics.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
```suggestion
// Copyright 2024 Filip Strzałka (@philvec)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
ok, I checked that it complies with the docs recommended format and it does - I will also update to GPL v.3 for compliance with ChibIOS.